### DISCUSSION TITLE
2.3.10 Installation Fail

### DISCUSSION THREAD

**Discussion by cybersimplified** [Security-Onion-Solutions/securityonion#2224] [Unsupported Versions] [answered]
Hi everyone, Everytime I use this github link to download the latest version iso image I get a "download fail" error after a couple minutes into the download. I'm using a Windows 10 Home OS with 12 gb ram & 500 gb SSD and yes I do have enough memory for the download. Not sure what else is causing this download to fail. I was able to download the file yesterday but I ended up uninstalling it because the installation on my Virtual Box kept looping to reboot. Then I went to reinstall and now I get this error. Thanks in advance.

**Comment by weslambert** [comment]
@npbarnes47 Have you tried it recently?  It's working fine for me.

**Comment by cybersimplified** [comment]
I was able to install it with vmware however now I'm stuck in the
installation setup & I'm not sure how to get out of the cli interface

**Reply by defensivedepth** [reply]
@npbarnes47 Have you seen our free Essentials training? It walks you through step-by-step on how to run through the install:

https://www.youtube.com/watch?v=BFdTHGcsmkk&list=PLljFlTO9rB155aYBjHw2InKkSMLuhWpxH&index=2

**Comment by cybersimplified** [comment]
Thanks for the feedback, I have followed through the steps but I guess
where I'm lost is how the instructor knew which IP addresses to use/how to
enable analyst mode for desktop. Please correct me if I'm wrong but it
seems in this latest version of Security Onion that the OS works through a
web portal; I was expecting to download a typical desktop OS.

**Comment by cybersimplified** [comment]
I'm just trying to get the actual GUI for Security OS, I have the latest
ISO image file downloaded from github & I'm using vmware to create my vm
for Security Onion. All the hardware specs have been exceeded. After
downloading the OS I'm just stuck in the cli. I'm really confused of how to
access the GUI. In any help would be very much appreciated.

**Comment by dougburks** [comment, marked as answer]
If you'd like to install the optional desktop interface, please see https://docs.securityonion.net/en/2.3/analyst-vm.html.

**Comment by cybersimplified** [comment]
Thanks for following up. I have ran: sudo so-analyst-install     then the
vm I have for Security Onion reboots back to the same command line
interface; it seems the previous versions of Security Onion go right into a
GUI for installation I'm still confused on how to access the GUI from my VM

**Reply by dougburks** [reply]
When you ran `so-analyst-install`, did you have full Internet access?

Did you receive any error messages?

Are you able to duplicate this on a fresh install?

**Comment by cybersimplified** [comment]
I just ran this, and I do not have internet access to reach
https://securityonionsolutions.com/
I'm going to redownload the iso from
https://github.com/Security-Onion-Solutions/securityonion/blob/master/VERIFY_ISO.md
& try a fresh install    I'm going follow the eval mode once again as shown
in the tutorial videos, however eval mode won't actually provide
me with the OS for offline use or? This is partly where I get confused with
the difference between eval & analyst installs.

**Comment by cybersimplified** [comment]
Also when trying to set up analyst mode how much hard drive space should I
devote to the install? I gave my initial install 100gb / 20gb ram / 4
processors

**Comment by cybersimplified** [comment]
I tried verifying the image & I get this output when I run following command
$ gpg --verify securityonion-2.3.10.iso.sig securityonion-2.3.10.iso
gpg: Signature made Thu, Nov 19, 2020 12:38:54 PM PST
gpg:                using RSA key 7C1060B7FE507013
gpg: Can't check signature: No public key   <

I noticed on the walkthrough the instructor was able to check the public
key; I did download from github site

**Comment by cybersimplified** [comment]
So sorry for the many emails this morning but I have realized another issue
I am having is I need to free up another 100gb of space for a total of
200gb. Thanks again for all your help so far, I really appreciate it.
